Src/WindowsServer/WindowsServer.Shared.Tests/WindowsServer.Shared.Tests.projitems

```diff
@@ -21,10 +21,14 @@
     <Compile Include="$(MSBuildThisFileDirectory)DeveloperModeWithDebuggerAttachedTelemetryModuleTest.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)DeviceTelemetryInitializerTest.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Implementation\ServiceRuntimeTests.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Implementation\TypeHelpersTests.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Implementation\WindowsServerEventSourceTests.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)FirstChanceExceptionStatisticsTelemetryModuleTest.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)UnhandledExceptionTelemetryModuleTest.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)UnobservedExceptionTelemetryModuleTest.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <Content Include="$(MSBuildThisFileDirectory)Newtonsoft.Json.dll">
```

Comment:
nothing spl about this dll - testing the ability to load any given DLL from GAC to a new app domain - so i simply used a random (newtonsoft.json) dll.